ansible-deployment/ironic_config/tasks/main.yaml

```diff
@@ -20,5 +20,24 @@
          path: /var/lib/config-data/puppet-generated/ironic/etc/ironic/ironic.conf
          insertafter: ^# *socat_address *=.*
          line: socat_address = {{ public_ip }}
-     - name: restart ironic container
-       command: podman container restart ironic_conductor
+     - name: Add extra kernel params for inspection
+       lineinfile:
+         path: /var/lib/config-data/puppet-generated/ironic/etc/ironic/ironic.conf
+         insertafter: ^# *extra_kernel_params *=.*
+         line: extra_kernel_params = ipa-inspection-collectors=default,pci-devices
+     - name: Update inspection processing hooks
+       lineinfile:
+         path: /var/lib/config-data/puppet-generated/ironic_inspector/etc/ironic-inspector/inspector.conf
+         insertafter: ^# *processing_hooks *=.*
+         line: processing_hooks=$default_processing_hooks,extra_hardware,lldp_basic,local_link_connection,physnet_cidr_map,accelerators
```

Comment:
Do we need to care about the order of these hooks?

Reply:
Nope!